5-AdvancedScenarios/1-call-graph-bff/README.md

```diff
@@ -0,0 +1,255 @@
+
+1. Still on the same app registration, select the **Token configuration** blade to the left.
+1. Select **Add optional claim**:
+    1. Select **optional claim type**, then choose **ID**.
+    1. Select the optional claim **acct**.
+    > Provides user's account status in tenant. If the user is a **member** of the tenant, the value is *0*. If they're a **guest**, the value is *1*.
+    1. Select **Add** to save your changes.
+
+##### Configure the service app (msal-node-webapp) to use your app registration
+
+Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
+
+> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `APP\authConfig.js` file.
+1. Find the key `Enter_the_Application_Id_Here` and replace the existing value with the application ID (clientId) of `msal-node-webapp` app copied from the Azure portal.
+1. Find the key `Enter_the_Tenant_Id_Here` and replace the existing value with your Azure AD tenant/directory ID.
+1. Find the key `Enter_the_Client_Secret_Here` and replace the existing value with the generated secret that you saved during the creation of `msal-node-webapp` copied from the Azure portal.
+
+### Step 4: Running the sample
+
+```console
+    cd 5-AdvancedScenarios\1-call-graph-bff\App
+    npm run build
+    npm start
+```
+
+## Explore the sample
+
+1. Open your browser and navigate to `http://localhost:4000`.
+1. Select the **Sign In** button on the top right corner.
+1. Select the **Profile** button on the navigation bar. This will make a call to the Graph API.
+
+![Screenshot](./ReadmeFiles/screenshot.png)
+
+> :information_source: Did the sample not work for you as expected? Then please reach out to us using the [GitHub Issues](../../../../issues) page.
+
+## We'd love your feedback!
+
+Were we successful in addressing your learning objective? Consider taking a moment to [share your experience with us](Enter_Survey_Form_Link).
+
+
+## Troubleshooting
+
+<details>
+	<summary>Expand for troubleshooting info</summary>
+
+> * Use [Stack Overflow](http://stackoverflow.com/questions/tagged/msal) to get support from the community. Ask your questions on Stack Overflow first and browse existing issues to see if someone has asked your question before.
+Ask your questions on Stack Overflow first and browse existing issues to see if someone has asked your question before.
+Make sure that your questions or comments are tagged with [`azure-active-directory-b2c` `node` `ms-identity` `adal` `msal-js` `msal`].
+
+To provide feedback on or suggest features for Azure Active Directory, visit [User Voice page](https://feedback.azure.com/d365community/forum/79b1327d-d925-ec11-b6e6-000d3a4f06a4).
+</details>
+
+## About the code
+
+> * Describe where the code uses auth libraries, or calls the graph
+> * Describe specific aspects (e.g. caching, validation etc.)
+
+## Next Steps
+
+Learn how to:
+
+> * Enter next steps (samples, docs) for your platform here
```

Comment:
Would be nice to link to Hybrid SPA sample here

Reply:
Added!